contracts/Depository.sol

```diff
@@ -65,10 +65,11 @@ contract Depository is IErrorsTokenomics {
     event TreasuryUpdated(address indexed treasury);
     event BondCalculatorUpdated(address indexed _bondCalculator);
     event CreateBond(address indexed token, uint256 indexed productId, address indexed owner, uint256 bondId,
-        uint256 amountOLAS, uint256 tokenAmount, uint256 expiry);
-    event RedeemBond(uint256 indexed productId, address indexed owner, uint256 bondId);
-    event CreateProduct(address indexed token, uint256 indexed productId, uint256 supply, uint256 priceLP, uint256 expiry);
-    event CloseProduct(address indexed token, uint256 indexed productId);
+        uint256 amountOLAS, uint256 tokenAmount, uint256 maturity, uint256 startTime);
+    event RedeemBond(uint256 indexed productId, address indexed owner, uint256 bondId, uint256 redeemTime);
+    event CreateProduct(address indexed token, uint256 indexed productId, uint256 supply, uint256 priceLP,
+        uint256 vesting, uint256 startTime);
+    event CloseProduct(address indexed token, uint256 indexed productId, uint256 closeTime);
```

Comment:
Adding start and redeem / close time to events.

Reply:
Same as above

Reply:
Same as above